index.js

```diff
@@ -1547,7 +1552,7 @@
   //. Left('Cannot divide by zero')
   //. ```
   function Left(x) {
-    return new Either(sentinel, 'Left', x);
+    return new _Either('Left', x);
```

Comment:
I think `Left('x').constructor` would be `_Either` when by conforming FL it should be `Either`